sys-fs/gt5/gt5-1.5.0_pre20111220.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="a diff-capable 'du-browser'"
+HOMEPAGE="http://gt5.sourceforge.net/"
+SRC_URI="http://deb.debian.org/debian/pool/main/g/${PN}/${PN}_1.5.0~20111220+bzr29.orig.tar.gz -> ${P}.tar.gz"
```

Comment:
Do some substitution-foo to get this out of `${PV}`. I suppose appending `_p29` as a replacement for `bzr29` would be nice too.